skema/program_analysis/tests/test_fun_arg_fun_call.py

```diff
@@ -0,0 +1,323 @@
+# import json               NOTE: json and Path aren't used right now,
+# from pathlib import Path        but will be used in the future
+from skema.program_analysis.multi_file_ingester import process_file_system
+from skema.gromet.fn import GrometFNModuleCollection
+from skema.gromet.fn import FunctionType
+import ast
+
+from skema.program_analysis.CAST.pythonAST import py_ast_to_cast
+from skema.program_analysis.CAST2FN.model.cast import SourceRef
+from skema.program_analysis.CAST2FN import cast
+from skema.program_analysis.CAST2FN.cast import CAST
+from skema.program_analysis.run_ann_cast_pipeline import ann_cast_pipeline
+
+def fun_arg_fun_call():
+    return """
+def foo(x,y,z):
+    b = x(10) * 2
+    c = b * y(3)
+    a = x(z) + y(z)
+    return a
+
+def a(f): return f + 1
+def b(f): return f + 2
+
+foo(x=a,y=b,z=1)
+    """
+
+def generate_gromet(test_file_string):
```

Comment:
This is a minor thing, but since this function is defined in multiple test files, it may be useful to add it to `skema/utils`

Reply:
Right, there's a lot of repetitive things in the test cases that I would eventually like to add there. This is another good thing to add though.